README.md

```diff
@@ -39,3 +39,22 @@ const bucket2 = new Bucket(this, 'AnotherImageryArchive');
 applyTags(bucket1, commonTags);
 applyTags(bucket2, commonTags);
 ```
+## Contributing
+
+### Adding responder teams.
+
+Contact the prod ops team in slack at [#team-step-prod-ops](https://linz.enterprise.slack.com/archives/C05Q11EGLA0) 
+To create a new team in [opsgenie](https://toitutewhenua.app.opsgenie.com/teams/list)
+once this team is created, add a new responder team to `src/responder-teams.ts` file with the new team name. These two names 
+must match to ensure New Relic alerts are sent to the correct team.
+
+### Renaming responder teams.
```

Comment:
i almost feel we shoudl never do this.. 
if we still keep this section.. put a LOUD WARNING that says dont do this unless u know what you r doing and it is absolutely necessary.